ui/desktop/app/components/settings-card/client-agent/index.hbs

```diff
@@ -0,0 +1,72 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if this.isClientAgentActive}}
+  <SettingsCard
+    @header={{t 'settings.client-agent.title'}}
+    @icon='network-alt'
+    @description={{t 'settings.client-agent.description'}}
+  >
+    <:state>
+      {{#if this.isClientAgentStatusRunning}}
+        <Hds::Badge
+          @text={{t 'states.running'}}
+          @icon='running'
+          @color='success'
+        />
+      {{else}}
+        <Hds::Badge
+          @text={{t 'states.paused'}}
+          @icon='pause-circle'
+          @color='neutral'
+        />
+      {{/if}}
+    </:state>
+    <:body>
+      <div>
+        <Hds::Text::Display>
+          {{t 'settings.version'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{@model.clientAgentStatus.version}}
+        </Hds::Text::Body>
+      </div>
+      {{#if @model.clientAgentStatus.errors}}
+        <Hds::Alert @type='inline' @color='warning' as |A|>
+          <A.Title>{{t 'settings.alerts.client-agent'}}</A.Title>
+          <A.Description>
+            {{#each @model.clientAgentStatus.errors as |error|}}
+              <Hds::Text::Body @tag='p'>
+                {{error.message}}
+              </Hds::Text::Body>
+            {{/each}}
+          </A.Description>
+        </Hds::Alert>
+      {{/if}}
+
+    </:body>
+    <:action>
+      {{#if this.isClientAgentStatusRunning}}
+        <Hds::Button
+          @text={{t 'actions.pause'}}
+          @color='secondary'
+          @icon={{if (is-loading) 'loading' 'pause'}}
+          @size='medium'
+          disabled={{(is-loading)}}
+          {{on 'click' this.pauseClientAgent}}
+        />
+      {{else}}
+        <Hds::Button
+          @text={{t 'actions.resume'}}
+          @color='secondary'
+          @icon={{if (is-loading) 'loading' 'play'}}
+          @size='medium'
+          disabled={{(is-loading)}}
+          {{on 'click' this.resumeClientAgent}}
```

Comment:
nitpick: we can use the same action for both resume and pause as it is pretty much the same and pass in an arg

Reply:
I thought about that but decided against it since it doesn't really get us much besides not doing using two actions vs having an if check for a magic string. What I _could_ do is check what the current state of the agent is and then decide based on that, would that be better?

Reply:
yeah the state idea sounds much better